gto/api.py

```diff
@@ -53,10 +48,10 @@ def _is_gto_repo(repo: Union[str, Repo]):
 def _get_index(repo: Union[str, Repo], file=False):
```

Comment:
Is this function still used somewhere or it can be removed?

Reply:
good point, removed!